documents/Specification/MaterialX.Specification.md

```diff
@@ -1363,23 +1323,6 @@ Math nodes have one or two spatially-varying inputs, and are used to perform a m
     * `in1` (float or color<em>N</em> or vector<em>N</em>): the first value or nodename
     * `in2` (same type as `in1` or float): the second value or nodename
 
-<a id="node-and"> </a>
```

Comment:
https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/stdlib/stdlib_defs.mtlx#L4766

These nodes are now part of the implementation, along with the additional `xor` for completeness.

Reply:
Note that previously, the "and", "or" and "not" nodes were documented *twice* in the Spec, so I removed one set (from the "math nodes" section"), and kept the other (in a new Logical Operators section).